airbyte-integrations/connectors/destination-snowflake/src/main/java/io/airbyte/integrations/destination/snowflake/SnowflakeGcsStagingSqlOperations.java

```diff
@@ -106,9 +106,21 @@ public String uploadRecordsToStage(final JdbcDatabase database, final Serializab
         exceptionsThrown.add(e);
       }
     }
+    LOGGER.info("Successfully loaded records to stage {} with {} tries", stagingPath, exceptionsThrown.size());
```

Comment:
Since the normal exit is on line 103 above, do you want logging there as well?

Reply:
Good catch, this line doesn't get executed when successful records are successfully uploaded 🤦‍♂️ . It should be something like

```java
    while (exceptionsThrown.size() < UPLOAD_RETRY_LIMIT) {
      try {
        final String fileName = loadDataIntoBucket(stagingPath, recordsData);
        LOGGER.info("Successfully loaded records to stage {} with {} tries", stagingPath, exceptionsThrown.size());
        return fileName;
      } catch (final Exception e) {
        LOGGER.error("Failed to upload records into storage {}", stagingPath, e);
        exceptionsThrown.add(e);
      }
    }
```